docs/topics/signals.rst

```diff
@@ -158,6 +158,23 @@ item_error
     :param failure: the exception raised
     :type failure: twisted.python.failure.Failure
 
+bytes_received
+--------------
+
+.. signal:: bytes_received
+.. function:: bytes_received(data, request)
```

Comment:
Could you please add spider argument to this signal as well, like for other signals?